tests/payloads/github.json

```diff
@@ -4,9 +4,9 @@
   "pull_request": {
     "url": "https://api.github.com/repos/edx/openedx-webhooks/pulls/82",
```

Comment:
It kind of doesn't matter, since these .json files are just examples saved for us to use while developing, but here's an `/edx/` in the api.github.com URL.  There are other `/edx/` throughout the JSON.

But I wonder if there are other api.github.com URLs that need to be fixed in other repos where they are really used for something?

Reply:
HMM yes. Probably. Time for another set of prs 😆 😱 

Makes sense to change them here, at least.

Reply:
Looking through the repos, it looks like the only GitHub api URLs that need fixing are in repo-health.